api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -70,10 +70,18 @@ public static class BillingConfig {
     // The free tier GCP billing account ID to associate with Terra / GCP projects.
     public String accountId;
 
+    // The legacy free tier billing account id that is migrating away. This value helps to make
+    // migration process smooth.
+    public String legacyAccountId;
+
     public String freeTierBillingAccountName() {
       return "billingAccounts/" + accountId;
     }
 
+    public String legacyFreeTierBillingAccountName() {
```

Comment:
nit: perhaps add a helper here like `Set<String> freeTierBillingAccountNames()`

should make the two comparison code sites much easier